include/zenoh-pico/api/types.h

```diff
@@ -197,45 +190,42 @@ _OWNED_TYPE_PTR(_z_encoding_t, encoding)
 _LOANED_TYPE(_z_encoding_t, encoding)
 
 /*
- * Represents timestamp value in Zenoh
+ * Represent timestamp value in Zenoh
  */
 typedef _z_timestamp_t z_timestamp_t;
 
 /**
- * Represents a Zenoh value.
+ * Represent a Zenoh value.
  *
  * Members:
  *   z_loaned_encoding_t encoding: The encoding of the `payload`.
  *   z_loaned_bytes_t* payload: The payload of this zenoh value.
  */
-
 _OWNED_TYPE_PTR(_z_value_t, value)
 _LOANED_TYPE(_z_value_t, value)
 
 /**
- * Represents the set of options that can be applied to a (push) subscriber,
- * upon its declaration via :c:func:`z_declare_subscriber`.
+ * Represent the configuration used to configure a subscriber upon declaration.
  *
  * Members:
- *   z_reliability_t reliability: The subscription reliability.
+ *   z_reliability_t reliability: The subscription reliability value.
  */
 typedef struct {
     z_reliability_t reliability;
 } z_subscriber_options_t;
 
 /**
- * Represents the replies consolidation to apply on replies to a :c:func:`z_get`.
+ * Represent the reply consolidation mode to apply on replies to a :c:func:`z_get`.
  *
  * Members:
- *   z_consolidation_mode_t mode: Defines the consolidation mode to apply to the replies.
+ *   z_consolidation_mode_t mode: the consolidation mode, see :c:type:`z_consolidation_mode_t`
  */
 typedef struct {
     z_consolidation_mode_t mode;
 } z_query_consolidation_t;
 
 /**
- * Represents the set of options that can be applied to a publisher,
```

Comment:
What was the reason to change it?
IMHO main goal of this PR should be a correction for irrelevant comments of those API methods that have been changed.